src/main/java/com/google/sps/data/User.java

```diff
@@ -1,11 +1,15 @@
 package com.google.sps.data;
 
+import java.util.ArrayList;
+
 public final class User {
 
   private final String name;
+  private final ArrayList<String> productArea;
 
   public User(String name) {
     this.name = name;
+    productArea = new ArrayList<String>();
```

Comment:
Will you need to store duplicates and does ordering matter? If not, consider using a Set instead of a List.

![image](https://user-images.githubusercontent.com/5056759/86273995-dfeb3680-bb85-11ea-97d0-25c5b881059c.png)

Reply:
Good point. I changed it to a HashSet because data should not be repeated and it is unordered.